# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/Navigator.cpp
# Commit: 46af55564501
# Full Hash: 46af555645013a126416a440b658f2969eca4b20
# Author: Benjamin VanderSloot <bvandersloot@mozilla.com>
# Date: 2025-06-06 08:52:03
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D251617
# ==============================================================================

diff -r 55c4afa3a46c -r 46af55564501 dom/base/Navigator.cpp
--- a/dom/base/Navigator.cpp	Thu Jun 05 14:53:36 2025 +0000
+++ b/dom/base/Navigator.cpp	Thu Jun 05 14:53:36 2025 +0000
@@ -2269,7 +2269,7 @@
 
 NavigatorLogin* Navigator::Login() {
   if (!mLogin) {
-    mLogin = new NavigatorLogin(GetWindow()->AsGlobal());
+    mLogin = new NavigatorLogin(GetWindow());
   }
   return mLogin;
 }